### ISSUE TITLE
Errno -13: Failed Install on macOS 

### ISSUE THREAD

**Issue by nicholasurban** [dragonwocky/notion-enhancer#125]
I'm running Catalina 10.15.6. When trying to install with `sudo npm i -g notion-enhancer` I get:

> === NOTION ENHANCEMENT LOG ===
>  ...unpacking app.asar.
> ### ERROR ###
> Error: EACCES: permission denied, mkdir '/Applications/Notion.app/Contents/Resources/app'
>     at Object.mkdirSync (fs.js:921:3)
>     at Object.promisified.mkdirpSync (/usr/local/lib/node_modules/notion-enhancer/node_modules/asar/lib/wrapped-fs.js:24:38)
>     at module.exports.extractAll (/usr/local/lib/node_modules/notion-enhancer/node_modules/asar/lib/asar.js:177:6)
>     at module.exports (/usr/local/lib/node_modules/notion-enhancer/pkg/apply.js:57:5)
>     at async CAC.<anonymous> (/usr/local/lib/node_modules/notion-enhancer/bin.js:29:5) {
>   errno: -13,
>   syscall: 'mkdir',
>   code: 'EACCES',
>   path: '/Applications/Notion.app/Contents/Resources/app'
> }

Since I ran the install command with `sudo` I'm surprised that it failed with a permission error.

**Comment by dragonwocky** [comment]
try running `chown -R /Applications/Notion.app/Contents/Resources`

**Comment by seltzdesign** [comment]
> try running `chown -R $USER /Applications/Notion.app/Contents/Resources`
> 
> even though you've run the installer with `sudo`, not all the scripts that come after it have elevated permissions. you'll get a more reliable result if you run without `sudo` and just give your user the necessary permissions.

Getting the same error and the suggestions sadly didn't help. Any other ideas?

**Comment by dragonwocky** [comment]
sure it's the same error? that shouldn't be possible...
what was the output of that `chown` command?

**Comment by rlrigdon** [comment]
I'm having the same problem. Initially got a permissions error, then tried the command with sudo. Then got the error listed above. When I did the chown command, nothing happened - it just took me to the next command line. I then checked the permissions for the directory shown and I have read write permissions. I even tried manually creating the /app directory under /Resources, and then got different permissions errors on individual files.
